en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,77 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signin) or use your Github account if you have one (see Github instructions below).
+3. Click _New Project_ and choose _Hello Webpage_
+4. Click Terminal button (on the left side of the window)
```

Comment:
I only see that button (or is it a menu entry?) when I click on "Tools" first.

Reply:
(Also, it looks like the terminal isn't needed until further below, where there's a new (and correct) instruction on how to open it anyway.)

Reply:
I have updated the instructions to match the latest Glitch user interface (Tools -> Terminal)

Reply:
I added more precise indications to find it.